express-notes-api/index.js

```diff
@@ -0,0 +1,114 @@
+const express = require('express');
+const fs = require('fs');
+
+const app = express();
+const expressJson = express.json();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express listening on port 3000.');
+});
+
+app.use(expressJson);
+
+let data;
+fs.readFile('data.json', 'utf-8', (err, datafile) => {
+  if (err) console.error(err);
+  data = JSON.parse(datafile);
+});
```

Comment:
This sets up what's known as a "race condition" where the server is "ready" before it's actually done loading the data into memory. I recommend just using `require()` at the top of your file here to bring in the data.